### PR TITLE
update index

### PR DESCRIPTION
**Pull request by xbankingorg**
Name: XBANKING
Twitter Link: https://x.com/strongbuycrypto
List of audit links if any: -
Website Link: https://xbanking.org/
Logo: https://xbanking.org/logo.png
Current TVL: $190.430.000 TVL
Treasury Addresses (if the protocol has treasury): No
Chain: No
Coingecko ID: https://www.coingecko.com/en/coins/xbanking
Coinmarketcap ID: No
Short Description (to be shown on DefiLlama): Non-custodial staking & liquid pools provider.
Token address and ticker if any: 0x33E80a92a9Ea73dD02f6e732D1702d58c68388cA XB
Category: Staking Pool
Oracle used: Own
forkedFrom: No
Github org/user: https://github.com/xbankingorg

### CODE REVIEW

File: projects/xbanking/index.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
+const sdk = require('@defillama/sdk');
+const ADDRESSES = require('../helper/coreAssets.json')
+const ETHEREUM_CONTRACT = "0xC4356aF40cc379b15925Fc8C21e52c00F474e8e9";
+
+
+async function tvl(_, _1, _2, { api }) {
+  const bal = await api.call({
+    abi: 'function getTvl() external view returns (uint256)',
```
Comment (g1nt0ki) [inline projects/xbanking/index.js:8]: I have no clue what is going on here/why the number returned here should be taken as Tvl.

Can you give a token breakdown/where the tokens are locked?

Comment (xbankingorg) [reply projects/xbanking/index.js:8]: how I can test this code?

Comment (g1nt0ki) [reply projects/xbanking/index.js:8]: Try running "node test.js projects/xbanking"

Comment (xbankingorg) [reply projects/xbanking/index.js:8]: where I need to run that?

File: .github/workflows/main.yml
Comment (g1nt0ki) [inline .github/workflows/main.yml:1]: Please remove this file, it is not the one causing issue. My comment was for you test on your local machine. It is not run on GitHub as your pr has merge conflicts

**Review by g1nt0ki — CHANGES_REQUESTED** [review]
I don't understand what is being tracked as Tvl, please provide more details

**Comment by xbankingorg** [comment]
Name: XBANKING
Twitter Link: https://x.com/xbanking_org
List of audit links if any: -
Website Link: https://xbanking.org/
Logo: https://xbanking.org/logo.png
Current TVL: $190.430.000 TVL
Treasury Addresses (if the protocol has treasury): No
Chain: No
Coingecko ID: https://www.coingecko.com/en/coins/xbanking
Coinmarketcap ID: No
Short Description (to be shown on DefiLlama): Non-custodial staking & liquid pools provider.
Token address and ticker if any: 2uAuGwYH22SJJtaTqMJ2AGEL2rBdiRKkuak2QCCSaFCAXB
Category: Staking Pool
Oracle used: Own
forkedFrom: No
Github org/user: https://github.com/xbankingorg

**Pull request closed without merging**